generator/internal/client_generator.cc

```diff
@@ -78,8 +78,8 @@ Status ClientGenerator::GenerateHeader() {
                                   : vars("connection_rest_header_path"),
        IsExperimental() ? "google/cloud/experimental_tag.h" : "",
        "google/cloud/future.h", "google/cloud/options.h",
-       "google/cloud/polling_policy.h", "google/cloud/status_or.h",
-       "google/cloud/version.h"});
+       "google/cloud/polling_policy.h", "google/cloud/internal/make_status.h",
+       "google/cloud/status_or.h", "google/cloud/version.h"});
```

Comment:
Maybe this should be:

```suggestion
       "google/cloud/polling_policy.h",
         (get_iam_policy_extension_ && set_iam_policy_extension_ ) ? 
 "google/cloud/internal/make_status.h" : "",
       "google/cloud/status_or.h", "google/cloud/version.h"});
```

Adding a helper to say `HasIamPolicyExtension()` would be cleaner too.

Reply:
Good catch. Done!